app/src/main/java/org/mozilla/reference/browser/BrowserApplication.kt

```diff
@@ -20,6 +20,7 @@ import org.mozilla.reference.browser.ext.components
 import org.mozilla.reference.browser.ext.isCrashReportActive
 import org.mozilla.reference.browser.settings.Settings
 import org.mozilla.reference.browser.telemetry.GleanFactProcessor
+import mozilla.appservices.ReferenceBrowserMegazord
```

Comment:
We access this class via reflection APIs below, is this only here for testing in the same way as Nick mentions in https://github.com/mozilla-mobile/fenix/pull/1787?